src/index.html

```diff
@@ -5,8 +5,288 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <div class="container">
+        <div class="logo">
+          <a class="logo__link" href="#home">
+            <img class="logo__image"
+            src="./images/logo.png" alt="logo">
```

Comment:
format according to [line-wrapping](https://mate-academy.github.io/style-guides/htmlcss.html#html-line-wrapping)

Reply:
@aureliagorzna please update it accordingly ⬆️